### PR TITLE
User manual: updates to undo description

### PR DESCRIPTION
**Pull request by dirkhh**
And some trailing space removal.

The text on editing dives hopefully is more accurate and easier to understand.

### Describe the pull request:

- [ ] Bug fix
- [ ] Functional change
- [ ] New feature
- [ ] Code cleanup
- [ ] Build system change
- [X] Documentation change
- [ ] Language translation

### Mentions:

@willemferguson I hope this doesn't mess with the changes you had waiting - I notice your last one was "2 of 3". So if you'd rather see me apply "3 of 3", first, that's fine. Please submit it, though :-)

I notice that there are many more references to `Apply changes` in the user manual, including in many pictures. I had badly underestimated how much work was left in the manual...

### CODE REVIEW

**Comment by dirkhh** [comment]
OK, this is clean now. @willemferguson I'd appreciate a comment from you on whether this is the way to go... and a timeline for your "3 of 3" changes...

File: Documentation/user-manual.txt [hunk 1]
```diff
@@ -229,24 +229,25 @@ of each dive. These panels are respectively marked [red]#A#, [red]#B# and
...
 image::images/AddDive1_f22.jpg["FIGURE: Add dive",align="center"]
-When you edit a field in the Notes or Equipment panels, _Subsurface_ enters *Editing Mode*,
-indicated by the message in the blue box at the top of the *Notes* panel (see the image below). This message is
-displayed in all the panels under Notes and Equipment when in *Editing Mode*.
+
+Most actions when editing a new dive can be individually undone, as described in the
+section on <<S_undo,undoing edits>>, but changes
+in the Equipment panel are not yet included in that in-depth undo implementations, and therefore
+_Subsurface_ enters *Editing Mode* when changes are made in this panel. This is indicated
+indicated by the message in the blue box at the top of the panels (see the image below).
```
Comment (neolit123) [inline Documentation/user-manual.txt:237]: double `indicated`

**Review by neolit123 — COMMENTED** [review]
found one double word, but LGTM.

**Comment by willemferguson** [comment]
Dirk, I am much more more than happy with these changes. PR # 3 of 3 was supposed to be a part of what you have just done, with a few image updates. I also underestimated the changes required in the text. If these changes can be pulled into master I will submit PR #3 of 3.

**Pull request merged**